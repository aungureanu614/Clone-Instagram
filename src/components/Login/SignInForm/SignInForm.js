import React from 'react';

const SignInForm = () => (
  <div>
    Hi from sign in form
  </div>
);

export default SignInForm;
