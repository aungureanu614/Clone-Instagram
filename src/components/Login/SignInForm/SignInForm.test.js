import React from 'react'
import renderer from 'react-test-renderer'
import SignInForm from './SignInForm'

test('LoginContainer renders', () => {
  const component = renderer.create(<SignInForm />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
