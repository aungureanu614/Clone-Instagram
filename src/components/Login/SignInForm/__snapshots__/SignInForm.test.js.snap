// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginContainer renders 1`] = `
<div>
  Hi from sign in form
</div>
`;
